lib/matplotlib/axes/_axes.py

```diff
@@ -2858,7 +2866,7 @@ def errorbar(self, x, y, yerr=None, xerr=None,
         else:
             xuplims = np.asarray(xuplims, bool)
 
-        everymask = np.arange(len(x)) % errorevery == 0
+        everymask = np.arange(len(x)) % errorevery == offset
```

Comment:
That is a good catch.

We should either validate that offset < markevery or set other = other %
markevery and should probably cast both to int just to be safe.

On Fri, Apr 8, 2016, 18:03 Elliott Sales de Andrade <
notifications@github.com> wrote:

> In lib/matplotlib/axes/_axes.py
> <https://github.com/matplotlib/matplotlib/pull/6280#discussion_r59100637>:
>
> > @@ -2858,7 +2866,7 @@ def errorbar(self, x, y, yerr=None, xerr=None,
> >          else:
> >              xuplims = np.asarray(xuplims, bool)
> >
> > -        everymask = np.arange(len(x)) % errorevery == 0
> > +        everymask = np.arange(len(x)) % errorevery == offset
>
> This is not quite what the documentation states above (although the what's
> new does say it more explicitly.) Consider errorevery = 2, offset = 2.
> Nothing in the above documentation states that it won't work, but
> everymask will be all False here.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/6280/files/a9c05fb05bb95e93ced6349706b2f497fa731b08#r59100637>
>